meshnet/README.md

```diff
@@ -0,0 +1,89 @@
+Meshnet @ DWeb Camp 2019
+========================
+
+The 2019 Decentralized Web ~Summit~ Camp will take place at a beautiful piece of coastal land 40 minutes west of San Jose from July 18 to 21.
+This area currently has very minimal network infrastructure and Internet connectivity.
+Let's use this opportunity to build, test, and use our decentralized systems together,
+in a connectivity environment that more closely resembles large parts of the world,
+and co-imagine the social processes and local technologies that will help us organize day-to-day,
+and make inclusive spaces to learn from each other and build capacity around human-empowering technologies.
+
+![access-point-positions](images/access-point-positions.png?raw=true)
+
+## Goals
+
+This document is the proposal to build a local mesh network with the following goals in mind:
+
+- Reliable local network with 802.11ac speeds
+- Access points in key areas for client devices (e.g. phones and laptops) to connect
+- Allow Camp participants to extend or run locally-hosted services on the network
+- Carry traffic to Internet gateway for client devices to access the Internet with limited bandwidth and reliability
+- Document setup and learnings for future events or other groups
+
+## Hardware
+
+The diagram above is the area map, with preliminary positions of _mesh nodes_ that together form the local mesh network.
+Each node has a 100 m radius circle around it, representing the region where client devices are expected to form a good connection with the node's access point.
+There is a larger 150 m radiu circle around where devices may be able to connect. The green dotted lines represent point-to-point links between nodes that together form the local mesh network.
+
+Each node is an aggregate of multiple pieces of equipment, which may include:
+
+- Radio for directional link
+- Radio for access point
+- Network switch, router, and application server
+- Power equipment, mounts, and cables
+
+Some nodes will have interfaces that allow other devices to be plugged in to offer locally-hosted services on the network. 
```

Comment:
Will/should I be able to host a service on the network from my laptop that is connected to the network?

Or will I need to bring a microcomputer (e.g. rpi) and plug into one of these official planned nodes?

Reply:
Yes but IP allocation hasn't been well thought out. Adding to this https://github.com/dweb-camp-2019/organizing/issues/17